share/examples/bitcoin.conf

```diff
@@ -66,6 +66,10 @@
 # Maximum number of inbound+outbound connections.
 #maxconnections=
 
+#Maximum daily upload target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB max throughput per day)
+#Limiting the free-for-all bandwidth uploads for those with bandwidth limits
+#maxuploadtarget=
```

Comment:
suggestion
```diff
-#Maximum daily upload target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB max throughput per day)
-#Limiting the free-for-all bandwidth uploads for those with bandwidth limits
+# Maximum upload bandwidth limit in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB per day).
+# This limits the upload bandwidth for those with bandwidth limits. 0 = no limit (default: 0).
```

- "daily upload target  in MiB per day" is redundant (daily/per day)
- it's a limit, not a target
- mention the default and that 0 = no limit

Reply:
maybe mention that it does not apply to peers with "download" permission
```
$ bitcoind -help | grep -A3 maxuploadtarget=
  -maxuploadtarget=<n>
       Tries to keep outbound traffic under the given target (in MiB per 24h).
       Limit does not apply to peers with 'download' permission. 0 = no
       limit (default: 0)
```

Reply:
Awesome! Will get to this shortly. I am having trouble squashing after I enabled 2FA and CLI won't allow me to authorize now to squash. I am also trying the Desktop version and does not give me a squash option, tried researching, any help appreciated.